RFC.md

```diff
@@ -0,0 +1,327 @@
+Glossary:
+
+These are terms that I think deserves an up-front definition in the spec:
+
+    - `signal`
+    - `signaller`
+    - `emit` / `send` <-- change spec to use one of these
+    - `fail` / `failure` / `failing`
+    - `demand`
+    - `item`/`data`/`element` <-- change spec to use one of these
+    - `upstream` / `downstream`
+    - `return normally`
+    - `terminal state` / `end-of-stream` <-- change spec to use one of these
+    - `responsivity` / `non-obstructing` <-- change spec to use one of these
+    - `call`/`invoke` <-- change spec to use one of these
+
```

Comment:
Possible Issues that this proposal would resolve:

https://github.com/reactive-streams/reactive-streams-jvm/issues/334
https://github.com/reactive-streams/reactive-streams-jvm/issues/328
https://github.com/reactive-streams/reactive-streams-jvm/issues/325
https://github.com/reactive-streams/reactive-streams-jvm/issues/324
https://github.com/reactive-streams/reactive-streams-jvm/issues/323
https://github.com/reactive-streams/reactive-streams-jvm/issues/322
https://github.com/reactive-streams/reactive-streams-jvm/issues/321
https://github.com/reactive-streams/reactive-streams-jvm/issues/319
https://github.com/reactive-streams/reactive-streams-jvm/issues/290